documentation-site/pages/components/slider.mdx

```diff
@@ -36,27 +35,26 @@ Each slider has a primary control handle that cues user interaction.
   <Basic />
 </Example>
 
-<Example title="Slider as a Stateful Component" path="examples/slider/stateful.js">
+<Example
+  title="Slider as a Stateful Component"
+  path="examples/slider/stateful.js"
+>
   <Stateful />
 </Example>
 
-<Example title="Slider Range example" path="examples/slider/range.js">
+<Example title="Slider Range" path="examples/slider/range.js">
   <Range />
 </Example>
 
-<Example
-  title="Slider with custom labels"
-  path="examples/slider/range-label.js"
->
-  <RangeLabel />
-</Example>
-
-<Example title="Slider as a stepped slider" path="examples/slider/step.js">
-  <Step />
+<Example title="Disabled Slider" path="examples/slider/disabled.js">
+  <Disabled />
 </Example>
 
-<Example title="Slider with Multiple Ticks" path="examples/slider/ticks.js">
```

Comment:
is this use-case still possible? 
<img width="745" alt="screen shot 2019-02-11 at 8 19 54 am" src="https://user-images.githubusercontent.com/5317799/52576953-f8628180-2dd5-11e9-88aa-6574c5a8adc3.png">

Reply:
answered in a previous comment